### ISSUE TITLE
Normal mode 's' key is broken in 1.12

### ISSUE THREAD

**Issue by dcervenkov** [VSCodeVim/Vim#4359]
**Describe the bug**
Since 1.12.1 (1.12.0 was not working at all), pressing `s` in normal mode does this:
1. Go to end of line
2. Switch to insert mode
3. Write `symotion-s2)`

**To Reproduce**
1.  Open any file
2.  Press `s` while in normal mode

**Expected behavior**
The symbol under cursor should be deleted and the editor should switch to insert mode. 

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: `1.12.1`, `1.12.2`
- VSCode version: `1.40.2`
- OS: `Linux x64 4.15.0-72-generic`

**Additional context**
For testing, I disabled all extensions except Vim and deleted `settings.json`.

**Comment by HawkinsT** [comment]
Just to report 's' (delete  character and enter insert mode) is still working fine for me under windows in 2.12.2. Does it always take you to the end of a line and not the end of the first word on the line? I'm assuming what's happening when you hit 's' is '(easymotion-s2)' is being sent, with (ea being interpreted in normal mode and the a switching the rest to insert mode.

**Comment by dcervenkov** [comment]
> I'm assuming what's happening when you hit 's' is '(easymotion-s2)' is being sent, with (ea being interpreted in normal mode and the a switching the rest to insert mode.

Yes, you are right. The `(` moves the cursor to the beginning of the *sentence*, which is usually a few lines above. This is exactly the behavior I observe. Testing it with a single line and word mislead me.

**Comment by J-Fields** [comment]
@dcervenkov would you mind listing your settings?

**Comment by hiroto78** [comment]
It seems like  'p'  key also is broken. When key 'p' typed, the cursor would blink many times and the pane would be frozen.

**Comment by J-Fields** [comment]
@hiroto78 Are you also on linux?

**Comment by hiroto78** [comment]
@J-Fields On Mac. It looks different matter with the problem that Normal mode 's' key is broken in 1.12.

FYI
- Extension (VsCodeVim) version: 1.12.2
- VSCode version: 1.40.2
- OS: Mac 10.13.6

**Comment by dcervenkov** [comment]
> @dcervenkov would you mind listing your settings?

Default settings - I deleted `settings.json` for the test. I also disabled all other extensions.

**Comment by damassi** [comment]
This is happening to me as well -- every 10th `cmd+s` it outputs this string, as if its a race condition. 

On OSX Catalina.

**Comment by CVVPK** [comment]
I've been able to reproduce this bug. It's coming from a conflicting vimrc config, in specific I found it's reproducible by having a line like this:

```
nmap s <Plug>(easymotion-s2) 
```

This is what I'm seeing on the console when I press `s` in normal mode.

```
Remapper: debug: normalModeKeyBindingsMap. match found. before=s. after=<Plug>,(,e,a,s,y,m,o,t,i,o,n,-,s,2,). command=undefined.`
```
Same as with #4368  commenting out the line on vimrc solves it, or setting `vim.vimrc.enable` to `false`. Hopefully something like #4443 can be added, that seems  like a great idea.
